plugins/hls-refactor-plugin/test/Main.hs

```diff
@@ -640,7 +639,10 @@ renameActionTests = testGroup "rename actions"
       doc <- createDoc "Testing.hs" "haskell" content
       _ <- waitForDiagnostics
       actionsOrCommands <- getCodeActions doc (Range (Position 3 12) (Position 3 20))
-      [fixTypo] <- pure [action | InR action@CodeAction{ _title = actionTitle } <- actionsOrCommands, "monus" `T.isInfixOf` actionTitle , "Replace" `T.isInfixOf` actionTitle]
+      [fixTypo] <- pure [action | InR action@CodeAction{ _title = actionTitle } <- actionsOrCommands
+                                , "monus" `T.isInfixOf` actionTitle
+                                , "Replace" `T.isInfixOf` actionTitle
+                        ]
```

Comment:
Perhaps it would make sense to introduce a `findCodeActionsByInfix` as well? Could simplify the code in three locations.

Reply:
It's not so easy, because in this case the two isInfixOf are combined as if by &&.
Whereas the findCodeActions' (used to define findCodeActionsByPrefix) is effectively using || when multiple "search" strings are specified.

Reply:
Ah, I did not see that! Good point, thanks!